### PR TITLE
html css popup

### PR DESCRIPTION
**Pull request by YuliiaHordiichuk**
_Complete the form below and remove this line._

### task name

[Demo](https://yuliiahordiichuk.github.io/projects/popup/) |
[Code base](https://github.com/YuliiaHordiichuk/YuliiaHordiichuk.github.io/tree/main/projects/popup)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/YuliiaHordiichuk/popup/css/style.css [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+    width: 15px; 
+}
+
+::-webkit-scrollbar-thumb {
+    background-color: rgba(128, 128, 128, 0.2);
+  }
+
+::-webkit-scrollbar-track {
+    color: transparent;
+}
```
Comment (smillims) [inline submissions/YuliiaHordiichuk/popup/css/style.css:216]: Add an empty line

File: submissions/YuliiaHordiichuk/popup/index.html [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+                                    <figure class="popup__fig">
+                                        <img class="popup__icon" src="icons/documents.png" alt="Documents">
+                                        <figcaption class="popup__text">Documents</figcaption>
+                                    </figure>
+                                </a>
+                            </li>
+                        </ul>
+                    
+                        <input class='popup__checkbox' type="checkbox" id="popup-checkbox"> 
+                        <label class="popup__label" for="popup-checkbox" tabindex="0">More</label>
```
Comment (smillims) [inline submissions/YuliiaHordiichuk/popup/index.html:131]: Add margin-right for your More button or remove margin-left.
Screenshot_114

**Review by smillims — COMMENTED** [review]
Hey, good work, really! But let's some improve it

File: submissions/YuliiaHordiichuk/popup/index.html (hunk 2 above)
Comment (YuliiaHordiichuk) [reply submissions/YuliiaHordiichuk/popup/index.html:131]: Done and done :)

**Comment by smillims** [comment]
Well done! Now wait for mentor
If there is no one for a long time, then request to [FE Questionarium | Kottans](https://t.me/joinchat/DmX0JAl-mh5W0jrWli8Ycw) that someone from mentors review your work.

**Review by lenadgit — CHANGES_REQUESTED** [review]
There is a space between the border and the button. Could you please make the button's width the same as the width of a popup ?
emptyspace

File: submissions/YuliiaHordiichuk/popup/css/style.css [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+.popup__checkbox:focus + .popup__label {
+    outline: none; 
+    border: 1px solid rgba(128, 128, 128, 0.5);
+    border-radius: 5px;
+}
+
+.nav__checkbox {
+    width: 37px; 
+    height: 37px; 
+
```
Comment (lenadgit) [inline submissions/YuliiaHordiichuk/popup/css/style.css:84]: the good practice - do not leave an empty line between rules. Could you please fix it for all cases?

**Review by lenadgit — CHANGES_REQUESTED** [review]

**Comment by lenadgit** [comment]
Good job!  Thank you for the submission. Feel free taking the next course module while you submission is being reviewed

File: submissions/YuliiaHordiichuk/popup/css/style.css [hunk 4]
```diff
@@ -0,0 +1,215 @@
...
+.nav__item:last-child {
+    margin-right: 0;
+}
+
+.nav__link {
+    display: flex; 
+    align-items: center;
+    justify-content: center;
+    padding: 5px; 
+
```
[the remark this answers is not in the document]
Comment (YuliiaHordiichuk) [reply submissions/YuliiaHordiichuk/popup/css/style.css]: Done

**Comment by YuliiaHordiichuk** [comment]
@lenadgit thanks, I`ve done changes :)

**Comment by YuliiaHordiichuk** [comment]
@MaxKalashnyk Hi! Could you check my work, please? I`ve done changes but the previous reviewer does not answer me for a long time :(

**Review by MaxKalashnyk — APPROVED** [review]
LGTM! Good job!

**Pull request merged**